### PR TITLE
ML-service deployment

### PR DESCRIPTION
**Pull request by sivakarthikeyanoptit**
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: ansible/roles/stack-sunbird/defaults/main.yml [hunk 1]
```diff
@@ -367,21 +367,21 @@ dial_liveness_readiness:
   healthcheck: true
   readinessProbe:
     httpGet:
-      path: /ping
+      path: /health
       port: 3000
-    initialDelaySeconds: 15
-    periodSeconds: 15
-    timeoutSeconds: 5
+    initialDelaySeconds: 30
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/defaults/main.yml:372]: Please don't change other service values.

File: kubernetes/ansible/roles/sunbird-deploy/tasks/main.yml [hunk 2]
```diff
@@ -38,13 +39,13 @@
   copy:
     dest: "{{ chart_path }}/keys/{{ adminutil_refresh_token_public_key_kid }}"
     content: "{{ core_vault_sunbird_sso_publickey }}"
-  when: release_name == "learner" or release_name == "lms" or release_name == "groups" or release_name == "knowledgemw"
+  when: release_name == "learner" or release_name == "lms" or release_name == "groups" or release_name == "knowledgemw"
```
Comment (keshavprasadms) [inline kubernetes/ansible/roles/sunbird-deploy/tasks/main.yml:42]: Remove the trailing space.

File: kubernetes/ansible/roles/sunbird-deploy/tasks/main.yml [hunk 3]
```diff
@@ -53,7 +54,7 @@
   include_role:
     name: mount-keys
     tasks_from: "{{item}}"
-  when: release_name == "learner" or release_name == "lms" or release_name == "groups" or release_name == "knowledgemw"
+  when: release_name == "learner" or release_name == "lms" or release_name == "groups"
```
Comment (keshavprasadms) [inline kubernetes/ansible/roles/sunbird-deploy/tasks/main.yml:57]: Don't change existing values.

File: ansible/roles/stack-sunbird/defaults/main.yml (hunk 1 above)
Comment (sivakarthikeyanoptit) [reply ansible/roles/stack-sunbird/defaults/main.yml:372]: done keshav

File: kubernetes/ansible/roles/sunbird-deploy/tasks/main.yml (hunk 2 above)
Comment (sivakarthikeyanoptit) [reply kubernetes/ansible/roles/sunbird-deploy/tasks/main.yml:42]: removed keshav

File: kubernetes/ansible/roles/sunbird-deploy/tasks/main.yml (hunk 3 above)
Comment (sivakarthikeyanoptit) [reply kubernetes/ansible/roles/sunbird-deploy/tasks/main.yml:57]: revited back the changes

File: ansible/roles/stack-sunbird/defaults/main.yml [hunk 4]
```diff
@@ -834,7 +834,6 @@ geo_location_city_ipv4: "{{ env_name }}_geo_location_city_ipv4"
 report_config: "{{ env_name }}_report_config"
 job_request: "{{ env_name }}_job_request"
 experiment_definition: "{{ env_name }}_experiment_definition"
-dataset_metadata: "{{ env_name }}_dataset_metadata"
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/defaults/main.yml:837]: Why are you removing this variable?

Comment (sivakarthikeyanoptit) [reply ansible/roles/stack-sunbird/defaults/main.yml]: added this variable

**Review by keshavprasadms — APPROVED** [review]

**Pull request merged**